ush/parsing_namelists_FV3.sh

```diff
@@ -216,6 +224,13 @@ if [ $CCPP_SUITE = "FV3_GSD_v0" ]; then
   bl_mynn_tkeadvect=${bl_mynn_tkeadvect:-".true."}	! In config.fcst
   bl_mynn_edmf_mom=${bl_mynn_edmf_mom:-"1"}	! In config.fcst
 EOF
+elif [ $CCPP_SUITE = "S2S" ]; then
+  cat >> input.nml << EOF
+  oz_phys      = ${oz_phys}             ! In config.fcst
+  oz_phys_2015 = ${oz_phys_2015}        ! In config.fcst
+  cplwav       = ${cplwav}              ! CROW configured
```

Comment:
cplwav and cplwav2atm are not CCPP_SUITE specific and are also valid for IPD, they should not be within an if statement.

Reply:
I will fix it. @JessicaMeixner-NOAA